x-pack/plugins/endpoint/server/routes/resolver/queries/children.test.ts

```diff
@@ -8,7 +8,7 @@ import { EndpointAppConstants } from '../../../../common/types';
 
 describe('children events query', () => {
   it('generates the correct legacy queries', () => {
-    const timestamp = new Date();
+    const timestamp = new Date().getTime();
```

Comment:
consider reverting this file

Reply:
going to keep it as is, the ChildrenQuery function uses the PaginationParams interface, which has timestamp as a number https://github.com/elastic/kibana/pull/58288/files/0626d65f70f9164285fce3659c7643625a2ee085#diff-530e696bd60363b2b701f88bbb9bc6feR14